### PR TITLE
added the dataservice MS with /health api and test

### PR DESCRIPTION
**Pull request by nemerna**
please check my changes!

### CODE REVIEW

**Comment by tmihalac** [comment]
Looks good

File: src/dataservice/pom.xml [hunk 1]
```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.redhat.depdraw</groupId>
+  <artifactId>dataservice</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <properties>
+    <compiler-plugin.version>3.8.1</compiler-plugin.version>
+    <maven.compiler.release>11</maven.compiler.release>
```
Comment (dmartinol) [inline src/dataservice/pom.xml:10]: Java 11 is fine, but we should try to be consistent at team's level to not create issues when someone else starts working on any of these projects

File: src/dataservice/pom.xml [hunk 2]
```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0"?>
```
Comment (dmartinol) [inline src/dataservice/pom.xml:1]: You also need to update the pom under src to also build the dataservice module (also during the CI build)

**Review by dmartinol — APPROVED** [review]

**Review by tmihalac — APPROVED** [review]

File: src/dataservice/pom.xml [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.redhat.depdraw</groupId>
```
[the remark this answers is not in the document]
Comment (nemerna) [reply src/dataservice/pom.xml:5]: fixed
referenced each other

**Pull request merged**